tests/integration/synapseutils/test_synapseutils_sync.py

```diff
@@ -105,51 +103,9 @@ def test_syncToSynapse():
                 orig_list = ['%s.%s' % (i.id, i.versionNumber) if isinstance(i, Entity) else i
                              for i in syn._convertProvenanceList(orig.split(';'))]
                 new_list = ['%s.%s' % (i.id, i.versionNumber) if isinstance(i, Entity) else i
-                             for i in syn._convertProvenanceList(new.split(';'))]
+                            for i in syn._convertProvenanceList(new.split(';'))]
                 assert_equals(set(orig_list), set(new_list))
 
-        
-def test_syncFromSynapse():
```

Comment:
This removed test is being shadowed by a subsequent (almost) identical test of the same name.